paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -1388,6 +1388,14 @@ class PaparazziPluginTest {
       .runFixture(fixtureRoot) { build() }
   }
 
+  @Test
+  fun scaledA11y() {
+    val fixtureRoot = File("src/test/projects/scaled-a11y")
+    gradleRunner
+      .withArguments("verifyPaparazziDebug", "--stacktrace")
+      .runFixture(fixtureRoot) { build() }
```

Comment:
nit: can we merge this into ComposeA11yTest + use unsafeUpdateConfig?  imo, it's more obvious/readable to keep all A11y tests in the same project.

Reply:
That suggestion sounds good to me, just updated this PR